rules/integrations/aws/privilege_escalation_aws_suspicious_saml_activity.toml

```diff
@@ -0,0 +1,71 @@
+[metadata]
+creation_date = "2021/09/22"
+maturity = "production"
+updated_date = "2021/09/22"
+integration = "aws"
+
+[rule]
+author = ["Austin Songer"]
+description = """
+Identifies when suspicious SAML activity has occurred in AWS. An adversary could gain backdoor access via SAML manipulation.
+"""
+false_positives = [
+    """
+    SAML Provider could be updated by a system administrator. Verify whether the user identity, user agent, and/or 
+    hostname should be making changes in your environment. SAML Provider being updated from unfamiliar users should 
+    be investigated. If known behavior is causing false positives, it can be exempted from the rule.
+    """,
+]
+from = "now-25m"
+index = ["filebeat-*", "logs-aws*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "AWS Suspicious SAML Activity"
+note = """## Config
+
+The AWS Fleet integration, Filebeat module, or similarly structured data is required to be compatible with this rule."""
+references = [
+    "https://docs.aws.amazon.com/IAM/latest/APIReference/API_UpdateSAMLProvider.html",
+    "https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRoleWithSAML.html",
+]
+risk_score = 21
+rule_id = "979729e7-0c52-4c4c-b71e-88103304a79f"
+severity = "low"
+tags = ["Elastic", "Cloud", "AWS", "Continuous Monitoring", "SecOps", "Identity and Access"]
+timestamp_override = "event.ingested"
+type = "query"
+
+query = '''
+event.dataset:aws.cloudtrail and event.provider:(iam.amazonaws.com or sts.amazonaws.com) and event.action:(Assumerolewithsaml or 
+UpdateSAMLProvider) and event.outcome:success
+'''
+
+[[rule.threat]]
+framework = "MITRE ATT&CK"
+[[rule.threat.technique]]
+id = "T1484"
+name = "Domain Policy Modification"
+reference = "https://attack.mitre.org/techniques/T1484/"
+[[rule.threat.technique.subtechnique]]
+id = "T1484.002"
+name = "Domain Trust Modification"
+reference = "https://attack.mitre.org/techniques/T1484/002/"
+
+
+[rule.threat.tactic]
+id = "TA0004"
+name = "Privilege Escalation"
+reference = "https://attack.mitre.org/tactics/TA0004/"
```

Comment:
I think https://attack.mitre.org/techniques/T1550/001/ and Defense Evasion would be a better fit here, we also need to rename the file